config_defaults_inc.php

```diff
@@ -2153,6 +2153,12 @@
  */
 $g_ldap_simulation_file_path = '';
 
+/**
+ * The LDAP fields of a user to be cached
+ * @global array $g_ldap_cache_fields
+**/
```

Comment:
Add extra space before the `**/`.

Reply:
`*` replaced by space to be conform with other comments